PlayFabSDK/source/PlayFabSettings.cs

```diff
@@ -38,7 +38,7 @@ public static string GetFullUrl(string apiCall)
             var baseUrl = UseDevelopmentEnvironment ? DevelopmentEnvironmentUrl : ProductionEnvironmentUrl;
             if (baseUrl.StartsWith("http"))
                 return baseUrl;
-            return "https://" + TitleId + baseUrl + apiCall;
+            return "https://" + TitleId + baseUrl + apiCall + "?sdk=" + SdkVersionString;
```

Comment:
>SdkVersionString [](start = 72, length = 16)

I am still not sure if we should send this with every each one call, instead of having, maybe, a special dedicated API method to gather some diagnostic info like that? But i don't have strong opinion, i see pros and cons here. Paul, what do you think?